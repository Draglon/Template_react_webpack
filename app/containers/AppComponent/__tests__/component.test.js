import React from 'react';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ConnectedAppComponent, { AppComponent } from '../component';

describe('<AppComponent /> component', () => {
  const initialState = {
    isLogged: false,
    removeSessionId: () => {},
  };
  let store;
  let component;

  beforeEach(() => {
    store = configureStore(initialState);
    component = shallow(<ConnectedAppComponent store={store} />);
  });

  it('Render property', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
