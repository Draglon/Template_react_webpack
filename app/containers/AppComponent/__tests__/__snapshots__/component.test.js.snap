// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppComponent /> component Render property 1`] = `
<BrowserRouter>
  <main
    className="main page"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </main>
</BrowserRouter>
`;
